libraries/classes/Controllers/Server/PrivilegesController.php

```diff
@@ -333,11 +333,11 @@ public function __invoke(ServerRequest $request): void
         if (
             $this->response->isAjax()
             && empty($_REQUEST['ajax_page_request'])
-            && ! isset($_GET['export'])
-            && (! isset($_POST['submit_mult']) || $_POST['submit_mult'] !== 'export')
-            && ((! isset($_GET['initial']) || $_GET['initial'] === '')
-                || (isset($_POST['delete']) && $_POST['delete'] === __('Go')))
-            && ! isset($_GET['showall'])
+            && ! $request->hasQueryParam('export')
+            && (! $request->hasBodyParam('submit_mult') || $request->getParsedBodyParam('submit_mult') !== 'export')
```

Comment:
```suggestion
            && $request->getParsedBodyParam('submit_mult') !== 'export'
```
Redundant. null !== 'export'